### ISSUE TITLE
Error if `.tissuumaps` folder doesn't exist

### ISSUE THREAD

**Issue by Bisaloo** [TissUUmaps/TissUUmaps#6]
## Issue

When trying to add an image layer, I get the following error in the terminal:

> Traceback (most recent call last):
  File "/home/sergio/.local/lib/python3.8/site-packages/tissuumaps/gui.py", line 848, in addLayer
    self.openImagePath(layerpath)
  File "/home/sergio/.local/lib/python3.8/site-packages/tissuumaps/gui.py", line 748, in openImagePath
    self.addRecent(folderpath)
  File "/home/sergio/.local/lib/python3.8/site-packages/tissuumaps/gui.py", line 438, in addRecent
    with open(recentFile, "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/sergio/.tissuumaps/recents.json'

### Environment

- ubuntu 20.04
- python 3.8.10
- TissUUmaps version: 3.0.9.1

## Diagnostic

This happens because the `.tissuumaps` folder doesn't exist and python therefore doesn't manage to create a `recents.json` file there. The problem is solved by manually creating a `.tissuumaps` folder in `$HOME`.

A simpler reproducible, self-contained, example of this issue is:

```py
with open("test_random_folder/test_random_file.txt", "w") as f:
  f.write("test")
```
> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'test_random_folder/test_random_file.txt'

But this works:

```py
os.mkdir("test_random_folder")
with open("test_random_folder/test_random_file.txt", "w") as f:
  f.write("test")
```
> 4

## Proposed fix

TissUUmaps should test if `.tissuumaps` exists, and create it if it doesn't, before trying to run any operation there.

**Comment by cavenel** [comment]
Hi,
Thank you for spotting this issue!

It should be fixed in version [3.0.9.3](https://github.com/TissUUmaps/TissUUmaps/releases/tag/3.0.9.3) (see https://github.com/TissUUmaps/TissUUmaps/commit/fe4d8fbc44b9a31a291a4b9aac47e70787780e34)

**Comment by Bisaloo** [comment]
Thanks for quick answer and the quick fix! :+1: I'll test the new version later today just in case but I don't anticipate any problems.
